### PR TITLE
Better messaging if runtime enable/disable fails

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Source/v2/Meadow.Cli/Commands/Current/Runtime/RuntimeDisableCommand.cs [hunk 1]
```diff
@@ -28,6 +28,11 @@ protected override async ValueTask ExecuteCommand()
 
             state = await device.IsRuntimeEnabled(CancellationToken);
 
+            if (state == true)
+            {
```
Comment (ctacke) [inline Source/v2/Meadow.Cli/Commands/Current/Runtime/RuntimeDisableCommand.cs:32]: Should throw a CommandException with this text

File: Source/v2/Meadow.Cli/Commands/Current/Runtime/RuntimeEnableCommand.cs [hunk 2]
```diff
@@ -28,6 +28,11 @@ protected override async ValueTask ExecuteCommand()
 
             state = await device.IsRuntimeEnabled(CancellationToken);
 
+            if (state == false)
+            {
+                Logger?.LogError("Failed to enable runtime");
```
Comment (ctacke) [inline Source/v2/Meadow.Cli/Commands/Current/Runtime/RuntimeEnableCommand.cs]: ibid

File: Source/v2/Meadow.Cli/Commands/Current/Runtime/RuntimeDisableCommand.cs (hunk 1 above)
Comment (adrianstevens) [reply Source/v2/Meadow.Cli/Commands/Current/Runtime/RuntimeDisableCommand.cs:32]: Good call - updated

**Review by ctacke — APPROVED** [review]

File: Source/v2/Meadow.Cli/Commands/Current/Runtime/RuntimeEnableCommand.cs (hunk 2 above)
Comment (adrianstevens) [reply Source/v2/Meadow.Cli/Commands/Current/Runtime/RuntimeEnableCommand.cs]: done

**Pull request merged**